clippy bug

```
make clippy 
   Compiling d-engine v0.1.3 (/Users/joshua/workspace/DEventLab/d-engine)
error: doc list item without indentation
 --> src/docs/architecture/election-design.md:5:1
  |
5 | If the leader receives an AppendEntries, RequestVote, or other RPC with a term number greater than its current term, it must ...
  | ^
  |
  = help: if this is supposed to be its own paragraph, add a blank line
  = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#doc_lazy_continuation
  = note: `-D clippy::doc-lazy-continuation` implied by `-D warnings`
  = help: to override `-D warnings` add `#[allow(clippy::doc_lazy_continuation)]`
help: indent this line
  |
5 |    If the leader receives an AppendEntries, RequestVote, or other RPC with a term number greater than its current term, it must step down and revert to a follower.
  | +++

error: doc list item without indentation
 --> src/docs/architecture/election-design.md:7:1
  |
7 | If the leader receives a RequestVote RPC and grants the vote to another candidate (usually because the candidate has a more u...
  | ^
  |
  = help: if this is supposed to be its own paragraph, add a blank line
  = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#doc_lazy_continuation
help: indent this line
  |
7 |    If the leader receives a RequestVote RPC and grants the vote to another candidate (usually because the candidate has a more up-to-date log), the leader must step down.
  | +++

error: doc list item without indentation
 --> src/docs/architecture/election-design.md:9:1
  |
9 | If the leader cannot reach a quorum (majority of nodes) for longer than one election timeout, it must assume it no longer hol...
  | ^
  |
  = help: if this is supposed to be its own paragraph, add a blank line
  = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#doc_lazy_continuation
help: indent this line
  |
9 |    If the leader cannot reach a quorum (majority of nodes) for longer than one election timeout, it must assume it no longer holds authority and step down.
  | +++

error: could not compile `d-engine` (lib) due to 3 previous errors
warning: build failed, waiting for other jobs to finish...
error: could not compile `d-engine` (lib test) due to 3 previous errors
make: *** [clippy] Error 101
```

--- a/src/docs/architecture/election-design.md
+++ b/src/docs/architecture/election-design.md
@@ -1,9 +1,10 @@
 ## Conditions That Cause a Leader to Step Down
 
 A leader will step down in the following situations:
-1.	Receives an RPC with a Higher Term
-If the leader receives an AppendEntries, RequestVote, or other RPC with a term number greater than its current term, it must step down and revert to a follower.
-2.	Grants a Vote to Another Candidate
-If the leader receives a RequestVote RPC and grants the vote to another candidate (usually because the candidate has a more up-to-date log), the leader must step down.
-3.	Fails to Contact the Majority for Too Long
-If the leader cannot reach a quorum (majority of nodes) for longer than one election timeout, it must assume it no longer holds authority and step down.
+
+1. Receives an RPC with a Higher Term
+   If the leader receives an AppendEntries, RequestVote, or other RPC with a term number greater than its current term, it must step down and revert to a follower.
+2. Grants a Vote to Another Candidate
+   If the leader receives a RequestVote RPC and grants the vote to another candidate (usually because the candidate has a more up-to-date log), the leader must step down.
+3. Fails to Contact the Majority for Too Long
+   If the leader cannot reach a quorum (majority of nodes) for longer than one election timeout, it must assume it no longer holds authority and step down.

